qa/pull-tester/rpc-tests.py

```diff
@@ -23,166 +23,144 @@
-                                 description=__doc__,
-                                 epilog='''
-Help text and arguments for individual test script:''',
-                                 formatter_class=argparse.RawTextHelpFormatter)
-parser.add_argument('--coverage', action='store_true', help='generate a basic coverage report for the RPC interface')
-parser.add_argument('--extended', action='store_true', help='run the extended test suite in addition to the basic tests')
-parser.add_argument('--help', '-h', '-?', action='store_true', help='print help text and exit')
-parser.add_argument('--nozmq', action='store_true', help='do not run the zmq tests')
-parser.add_argument('--parallel', type=int, default=4, help='how many test scripts to run in parallel. Default=4.')
-parser.add_argument('--win', action='store_true', help='signal that this is running in a Windows environment and that we should run the tests')
-(args, unknown_args) = parser.parse_known_args()
-
-#Create a set to store arguments and create the passon string
-tests = set()
-passon_args = []
-
-for arg in unknown_args:
-    if arg[:2] == "--":
-        passon_args.append(arg)
-    else:
-        tests.add(arg)
-
-BOLD = ("","")
-if os.name == 'posix':
-    # primitive formatting on supported
-    # terminal via ANSI escape sequences:
-    BOLD = ('\033[0m', '\033[1m')
-
-# Read config generated by configure.
-config = configparser.ConfigParser()
-config.read_file(open(os.path.dirname(__file__) + "/tests_config.ini"))
-
-ENABLE_WALLET = config["components"]["ENABLE_WALLET"] == "True"
-ENABLE_UTILS = config["components"]["ENABLE_UTILS"] == "True"
-ENABLE_BITCOIND = config["components"]["ENABLE_BITCOIND"] == "True"
-ENABLE_ZMQ = config["components"]["ENABLE_ZMQ"] == "True" and not args.nozmq
-
-RPC_TESTS_DIR = config["environment"]["SRCDIR"] + '/qa/rpc-tests/'
-
-print_help = args.help
-run_parallel = args.parallel
-
-#Set env vars
-if "BITCOIND" not in os.environ:
-    os.environ["BITCOIND"] = config["environment"]["BUILDDIR"] + '/src/bitcoind' + config["environment"]["EXEEXT"]
-
-if config["environment"]["EXEEXT"] == ".exe" and not args.win:
-    # https://github.com/bitcoin/bitcoin/commit/d52802551752140cf41f0d9a225a43e84404d3e9
-    # https://github.com/bitcoin/bitcoin/pull/5677#issuecomment-136646964
-    print("Win tests currently disabled by default.  Use -win option to enable")
-    sys.exit(0)
-
-if not (ENABLE_WALLET and ENABLE_UTILS and ENABLE_BITCOIND):
-    print("No rpc tests to run. Wallet, utils, and bitcoind must all be enabled")
-    sys.exit(0)
-
-# python3-zmq may not be installed. Handle this gracefully and with some helpful info
-if ENABLE_ZMQ:
-    try:
-        import zmq
-    except ImportError:
-        print("ERROR: \"import zmq\" failed. Use -nozmq to run without the ZMQ tests."
-              "To run zmq tests, see dependency info in /qa/README.md.")
-        raise
-
-BASE_SCRIPTS= [
-    # longest test should go first, to favor running tests in parallel
```

Comment:
Oops. This was a bad manual merge from a old branch.

Thanks for catching this. I've fixed this so #9276 is included.